eventlet/hubs/hub.py

```diff
@@ -21,9 +22,17 @@ def alarm_signal(seconds):
 
 from eventlet.hubs import timer, IOClosed
 from eventlet.support import greenlets as greenlet, clear_sys_exc_info
-import monotonic
+from eventlet import patcher
 import six
 
+mod = os.environ.get('EVENTLET_CLOCK')
+if not mod:
+    from monotonic import monotonic as eventlet_clock
+else:
+    mod = mod.rsplit('.', 1)
+    eventlet_clock = getattr(patcher.original(mod[0]), mod[1])
+del mod
```

Comment:
Putting all above in a function `def default_clock()` would fix the need for del.

Reply:
But then import in function is a disaster waiting to happen, so keep it here, sorry for noise.

Please rename `eventlet_clock` to `default_clock`. Everything in this project is implicitly `eventlet_` so this prefix does not help.